orchagent/portsorch.cpp

```diff
@@ -457,6 +457,7 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
         for (j = 0; j < attr.value.u32list.count; j++)
         {
             tmp_lane_set.insert(attr.value.u32list.list[j]);
+            m_availableLanes.insert(attr.value.u32list.list[j]);
```

Comment:
>m_availableLanes [](http://example.com/codeflow?start=12&length=16)

isn't m_availableLanes the same the same a tmp_lane_set, do we need to introduce this variable?

Reply:
I think we need this variable. Because tmp_lane_set was stored in the m_portListLaneMap, but m_portListLaneMap may be changed in line 2731 which will make us lose the real available lane set.
https://github.com/Azure/sonic-swss/blob/b6c62b9a9cf4c4b4ccc5ab82ddb9732c899f8dea/orchagent/portsorch.cpp#L2723-L2737